tests/data/bundles/test_quandl.py

```diff
@@ -184,46 +173,42 @@ def expected_dividend_adjustment(idx, symbol):
         return pricing, adjustments
 
     def test_bundle(self):
-        url_map = merge(
-            {
-                format_wiki_url(
-                    self.api_key,
-                    symbol,
-                    self.start_date,
-                    self.end_date,
-                ): test_resource_path('quandl_samples', symbol + '.csv.gz')
-                for symbol in self.symbols
-            },
-            {
-                format_metadata_url(self.api_key, n): test_resource_path(
+        with open(test_resource_path(
                     'quandl_samples',
-                    'metadata-%d.csv.gz' % n,
+                    'QUANDL_ARCHIVE.zip'), 'rb') as quandl_response:
+
+            self.responses.add(
+                self.responses.GET,
+                'https://file_url.mock.quandl',
+                body=quandl_response.read(),
+                content_type='application/zip',
+                status=200
+            )
```

Comment:
FWIW I don't think you need to indent all the rest of this, once you've read the `quandl_response`. You can close the file here, and leave the rest of the indentation unchanged.